lua/starfall/libs_cl/convar.lua

```diff
@@ -27,77 +27,77 @@ end
 
 
 --- Check if the given ConVar exists
--- @param name Name of the ConVar
--- @return True if exists
+-- @param string name Name of the ConVar
+-- @return boolean True if exists
 function convar_library.exists(name)
 	checkpermission(instance, nil, "convar")
 	checkluatype(name, TYPE_STRING)
 	return GetConVar(name)~=nil
 end
 
 --- Returns default value of the ConVar
--- @param name Name of the ConVar
--- @return Default value as a string
+-- @param string name Name of the ConVar
+-- @return string Default value as a string
 function convar_library.getDefault(name)
 	return getValidConVar(name):GetDefault()
 end
 
 --- Returns the minimum value of the convar
--- @param name Name of the ConVar
--- @return The minimum value or nil if not specified
+-- @param string name Name of the ConVar
+-- @return number The minimum value or nil if not specified
 function convar_library.getMin(name)
 	return getValidConVar(name):GetMin()
 end
 
 --- Returns the maximum value of the convar
--- @param name Name of the ConVar
--- @return The maximum value or nil if not specified
+-- @param string name Name of the ConVar
+-- @return number? The maximum value or nil if not specified
 function convar_library.getMax(name)
 	return getValidConVar(name):GetMax()
 end
 
 --- Returns value of the ConVar as a boolean.
 -- True for numeric ConVars with the value of 1, false otherwise.
--- @param name Name of the ConVar
--- @return The boolean value
+-- @param string name Name of the ConVar
+-- @return boolean The boolean value
 function convar_library.getBool(name)
 	return getValidConVar(name):GetBool()
 end
 
 --- Returns value of the ConVar as a whole number.
 -- Floats values will be floored.
--- @param name Name of the ConVar
--- @return The integer value or 0 if converting fails
+-- @param string name Name of the ConVar
+-- @return number The integer value or 0 if converting fails
 function convar_library.getInt(name)
 	return getValidConVar(name):GetInt()
 end
 
 --- Returns value of the ConVar as a floating-point number.
--- @param name Name of the ConVar
--- @return The float value or 0 if converting fails
+-- @param string name Name of the ConVar
+-- @return number? The float value or 0 if converting fails
```

Comment:
Stray question mark

Reply:
Ah nvm, seems question mark is optional return. I think it will always return number though.